doc/src/manual/documentation.md

```diff
@@ -579,15 +580,15 @@ Surround text that should be displayed exactly as written with single backticks,
 A paragraph containing a `literal` word.
 ```
 
-Literals should be used when writing text that refers to names of variables, functions, or other
-parts of a Julia program.
+Literals should be used when writing text that refers to the names of variables, functions,
```

Comment:
Inserting the article gives the wrong connotation to this sentence. `the` would be appropriate if what was meant was the names only (but not the underlying object), that isn't the case here, so the article should be omitted.

Reply:
```suggestion
Literals should be used when writing text that refers to names of variables, functions,
```